scripts/nested-libvirt/router-check.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+set -u
+set -o pipefail
+
+# TODO: add ability to pass cluster-name
+# wait for the router namespace
+KUBECONFIG="${HOME}"/clusters/nested/auth/kubeconfig
+API_UP=
+ROUTER_NAMESPACE=
+ROUTER_DEPLOYMENT=
+while true; do
+  if [[ ! -f "${KUBECONFIG}" ]]; then
+    sleep 15 & wait
+    continue
+  fi
+  if ! oc get nodes 2>/dev/null; then
+    echo "Waiting for API at $(oc whoami --show-server) to respond ..."
+    sleep 15 & wait
+    continue
+  elif [[ -z "${API_UP}" ]]; then
+    echo "API at $(oc whoami --show-server) has responded"
+    API_UP=1
+  fi
+  if [[ -z "${ROUTER_NAMESPACE}" ]]; then
+    # check multiple namespaces while we are transitioning to the new locations
+    if oc get deploy/router -n tectonic-ingress 2>/dev/null; then
+      ROUTER_NAMESPACE=tectonic-ingress
+      ROUTER_DEPLOYMENT="deploy/router"
+      elif oc get ds/router-default -n openshift-ingress 2>/dev/null; then
```

Comment:
Looks like you need to dedent this and some following lines.

Reply:
yup